dds/DCPS/XTypes/DynamicDataImpl.h

```diff
@@ -8,6 +8,7 @@
 
 #ifndef OPENDDS_SAFETY_PROFILE
 #  include "DynamicDataBase.h"
+#  include "DynamicDataXcdrReadImpl.h"
```

Comment:
Does this need an `#include` or is a forward declaration good enough?

Reply:
Yeah, forward declaration here and include in the cpp file should work.